### PR TITLE
Task 9.7

### PR DESCRIPTION
**Pull request by AdamPawlinski**
Game finished. Ask for approval.

### CODE REVIEW

File: js/script.js [hunk 1]
```diff
@@ -0,0 +1,126 @@
+var newGameBtn = document.getElementById('js-newGameButton');
```
Comment (matwjablonski) [inline js/script.js:1]: between different code sections add empty line - e.g. between functions

File: js/script.js [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+    computerPointsElem = document.getElementById('js-computerPoints');
+function newGame() {
+  player.name = prompt('Please enter your name', 'player name');
+  if (player.name) {
+    player.score = computer.score = 0;
+    gameState = 'started';
+    setGameElements();
+    playerNameElem.innerHTML = player.name;
+	setGamePoints();
+  }
```
Comment (matwjablonski) [inline js/script.js]: what will happen when use just click enter without typing a name?

File: js/script.js [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+    return possiblePicks[Math.floor(Math.random()*3)];
+}
+var playerPickElem = document.getElementById('js-playerPick'),
+    computerPickElem = document.getElementById('js-computerPick'),
+    playerResultElem = document.getElementById('js-playerResult'),
+    computerResultElem = document.getElementById('js-computerResult');
+function checkRoundWinner(playerPick, computerPick) {
+  playerResultElem.innerHTML = computerResultElem.innerHTML = '';
+  var winnerIs = 'player';
+    if (playerPick == computerPick) {
```
Comment (matwjablonski) [inline js/script.js]: always compare also type in javascript

File: scss/style.scss [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+  border-radius: 5%;
+  box-shadow: 3px 3px 2px #848487;
+  padding: 5px;
+  margin: 10px;
+  color: #FFF;
+}
+#js-newGameElement {
+  @extend %margin-top;
+}
+#js-playerPickElement,
```
Comment (matwjablonski) [inline scss/style.scss]: add empty line between css rules

**Review by matwjablonski — CHANGES_REQUESTED** [review]

**Comment by AdamPawlinski** [comment]
Bugs fixed. Please accept changes.

File: js/script.js [hunk 5]
```diff
@@ -49,39 +51,57 @@ function setGameElements() {
...
     playerNameElem.innerHTML = player.name;
-	setGamePoints();
+    setGamePoints();
   }
-}
+  if (player.name) {
+  start();
+  } else if(!player.name) {
+      player.name = prompt('You forgot to enter your name. Please tell me who you are?', 'player name');
+      if (player.name){
```
Comment (matwjablonski) [inline js/script.js]: too many nested if statements - you should nest your if only once :)

**Review by matwjablonski — CHANGES_REQUESTED** [review]

**Comment by AdamPawlinski** [comment]
less if statements. Hope that will be okey.

**Review by matwjablonski — APPROVED** [review]

**Pull request merged**